config/base/crds/kafka.banzaicloud.io_kafkaclusters.yaml

```diff
@@ -1126,6 +1182,11 @@ spec:
                   additionalProperties:
                     type: string
                   type: object
+                useExistingLBLoadBalancer:
+                  description: Set this to true if you dont want the operator to create
```

Comment:
don't*

Reply:
Unresolved conversation, typo still there

Reply:
My bad. I did the change, but didn't commit it.

Reply:
👍